packages/react-html/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-html",
+  "version": "19.0.0",
+  "private": true,
```

Comment:
Note that there's no flag here but the whole package is private so doesn't get publish once we land. We can land and iterate.

Reply:
Was worried this blocks it from being published in react-builds but it works: https://react-builds.vercel.app/api/prs/30105/packages/react-html